src/main/java/frc/trigon/robot/commands/commandfactories/GeneralCommands.java

```diff
@@ -22,12 +22,16 @@ public static Command withoutRequirements(Command command) {
     }
 
     /**
-     * @return a command that toggles between the SWERVE's default command, from field relative to self relative
+     * Creates a command that toggles between the swerve's default commands: joystick oriented rotation and normal rotation.
+     * Joystick oriented rotation is when the robot rotates directly to the angle of the joystick.
+     * Normal rotation is when the robot rotates at a speed depending on the power given to the joystick.
+     *
+     * @return the command
      */
-    public static Command getToggleFieldAndSelfRelativeDriveCommand() {
+    public static Command getToggleRotationModeCommand() {
```

Comment:
This command should say both kinds of rotation, like the previous command said both kinds of drive modes

Reply:
I think that clutters it unnecacerily. The jd expalins everything about the modes